pkg/virt-controller/watch/replicaset.go

```diff
@@ -341,7 +352,7 @@ func (c *VMIReplicaSet) scale(rs *virtv1.VirtualMachineInstanceReplicaSet, vmis
 // filterActiveVMIs takes a list of VMIs and returns all VMIs which are not in a final state
 func (c *VMIReplicaSet) filterActiveVMIs(vmis []*virtv1.VirtualMachineInstance) []*virtv1.VirtualMachineInstance {
 	return filter(vmis, func(vmi *virtv1.VirtualMachineInstance) bool {
-		return !vmi.IsFinal()
+		return !vmi.IsFinal() && vmi.DeletionTimestamp == nil
```

Comment:
Since that change is introduce here, it might make sense to copy the two tests from #1238 too.

Reply:
I think we can just rebase your PR after this one will be merged